kubernetes/omsagent.yaml

```diff
@@ -480,13 +480,13 @@ spec:
         rsName: "omsagent-rs"
       annotations:
         agentVersion: "1.10.0.1"
-        dockerProviderVersion: "10.0.0-1"
+        dockerProviderVersion: "10.0.0-3"
         schema-versions: "v1"
     spec:
       serviceAccountName: omsagent
       containers:
         - name: omsagent
-          image: "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod05262020"
```

Comment:
You need to update the agent version for telemetry - https://github.com/microsoft/Docker-Provider/blob/ci_dev/kubernetes/linux/Dockerfile

Reply:
> You need to update the agent version for telemetry - https://github.com/microsoft/Docker-Provider/blob/ci_dev/kubernetes/linux/Dockerfile

> good catch :). updated the dockerfile with latest image.

Reply:
Isn;t this parameterized now ?